test/fixtures/templates/bad/resources/rds/retention_period.yaml

```
@@ -13,9 +13,8 @@ Resources:
       DBSubnetGroupName: !Ref SubnetGroup
       DeleteAutomatedBackups: !Ref DeleteAutomatedBackups
       EnablePerformanceInsights: !Ref EnablePerformanceInsights
-      Engine: sqlserver-ee
+      Engine: aurora-postgresql
```

```suggestion
      Engine: sqlserver-ee
```

You can change this back.  This test is for validating if you have `BackupRetentionPeriod` configured when you aren't using aurora.

---

gotchya. I misinterpreted what the test was for. i searched tests for instances of the 'PerformanceInsightsRetentionPeriod' property and made the good match the bad except for the performance insight retention period i was submitting this PR for. i figured it would be too good to have my first PR go through without a hitch.